src/content/docs/extensions/iceberg.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: "iceberg extension"
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+## Usage
+
+The `iceberg` extension adds support for scanning and copying from [Apache Iceberg format](https://iceberg.apache.org/).
+
+The iceberg functionality is not available by default, so you would first need to install the iceberg
+extension by running the following commands:
+
+```sql
+INSTALL iceberg;
+LOAD EXTENSION iceberg;
+```
+
+### Example dataset
+
+To test the examples, download the [iceberg_data.zip](TODO: upload iceberg sample data for download) file and unzip it.
+
+### Common Functions
+
+#### Scan the Iceberg table
+```cypher
+LOAD FROM 'iceberg_table_path' (file_format='iceberg') RETURN *;
+```
+#### Copy the Iceberg table into a node table
+```cypher
+COPY table_name FROM 'iceberg_table_path' (file_format='iceberg');
+```
+#### Access the Iceberg Metadata
+```cypher
+CALL ICEBERG_METADATA('iceberg_table_path') RETURN *;
+```
+#### List the Iceberg Snapshots
+```cypher
+CALL ICEBERG_SNAPSHOTS('iceberg_table_path') RETURN *;
+```
+
+### Optional parameters
+
+| Parameter                     | Type      | Default      | Description                                                                     |
+|-------------------------------|-----------|--------------|---------------------------------------------------------------------------------|
+| allow_moved_paths             | BOOLEAN   | false        | Allows scanning Iceberg tables that are not located in their original directory |
+| metadata_compression_codec    | STRING    | ''           | Specifies the compression code used for the metadata files                      |
```

Comment:
@SterlingT3485 I think you mentioned the only supported compression codec is `gzip`? Should we state that here?

Reply:
Yes, let me add that in the table

Reply:
Added